.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,127 @@
+name: Docker build and Push
+
+# Trigger the workflow on pull request
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    # Checkout the repository to the GitHub Actions runner
+    - uses: actions/checkout@v3
+
+
+    # Set up Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Get branch name
+      id: branch_name
+      run: echo "::set-output name=branch::${GITHUB_HEAD_REF}"
+      
+    # Install yaml
+    - name: Install yaml and oyaml
+      run: |
+          pip install pyyaml
+          pip install oyaml
+
+    # Install linkml
+    - name: Install LinkML
+      run: pip install linkml
+
+    # Check if the PR description "Organization Name" has a value and matches the one in the model card
+    - name: Check PR description
+      if: github.event_name == 'pull_request' && github.event.pull_request.merged != true
+      id: check_pr_desc
+      run: |
+          python ./.github/workflows/check_pr_description.py
+      continue-on-error: true
+    
+    # Step 3: If the previous step failed, leave a comment and close the PR
+    - name: Leave a comment and close PR
+      if: steps.check_pr_desc.outcome != 'success' && github.event_name == 'pull_request' && github.event.pull_request.merged != true
+      run: |
+        comment_body="The PR description key 'Organization Name' is empty. Please fill in the required information."
+        
+        gh pr comment ${{ github.event.number }} --body "$comment_body"
+
+        gh pr close ${{ github.event.number }}
+
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    # Get the new model's folder name
+    - name: Get latest model folder
+      id: get_latest_folder
+      run: |
+          python ./.github/workflows/get_latest_model_name.py > latest_folder.txt
+    
+    # Read the new model's folder name and save it as an environment variable
+    - name: Read latest folder name
+      id: read_latest_folder
+      run: echo "model_folder=$(cat latest_folder.txt)" >> $GITHUB_ENV
+    
+    # Get the path to the Dockerfile
+    - name: Get Dockerfile path
+      id: get_dockerfile_path
+      run: |
+          python ./.github/workflows/get_dockerfile_path.py > dockerfile_path.txt
+    
+    # Read the path to the Dockerfile and save it as an environment variable
+    - name: Read Dockerfile path
+      id: read_dockerfile_path
+      run: echo "dockerfile_path=$(cat dockerfile_path.txt)" >> $GITHUB_ENV
+    
+    # Set the Docker image name
+    - name: Set Docker image name
+      id: set_image_name
+      run: |
+          echo "image_name=neuronets/${{ env.model_folder }}" >> $GITHUB_ENV
```

Comment:
Please refer to the previous comment about accessing output parameters. Consider replacing `env.model_folder` with `steps.read_latest_folder.outputs.model_folder`.